portfolio-jsm/src/components/Contact.tsx

```diff
@@ -1,106 +1,153 @@
-            height={isMobile ? "46px" : "56px"}
-            inputColor="bg-white-800 dark:bg-black-300 border-solid border-[1px] border-[#CCE1FF] dark:border-[#2C3C56]"
-          />
-          <h3 className="mt-4 text-18 dark:text-white-900 md:mt-6 md:text-20">
-            Write something about your project goals and timeframe
-          </h3>
-          <textarea
-            placeholder=""
-            onChange={() => {}}
-            className={`${isMobile ? "w-[327px]" : "w-[763px]"} h-[190px] ${
-              isMobile ? "px-4 py-2" : ""
-            } rounded-[10px] border-[1px] border-solid border-[#CCE1FF] bg-white-800 dark:border-[#2C3C56] dark:bg-black-300`}
-          ></textarea>
-          <h3 className="mt-4 text-18 dark:text-white-900 md:mt-6 md:text-20">
-            How to reach back to you? eg. phone number or email
-          </h3>
-          <div className="mb-6 md:mb-12">
+          <form ref={form} onSubmit={sendEmail}>
+            <h3 className="mt-4 text-18 dark:text-white-900 md:mt-6 md:text-20">
+              What is your name?
+            </h3>
             <Input
-              type="String"
+              type="text"
+              name="user_name"
               placeholder=""
               onChange={() => {}}
               width={isMobile ? "327px" : "763px"}
               height={isMobile ? "46px" : "56px"}
               inputColor="bg-white-800 dark:bg-black-300 border-solid border-[1px] border-[#CCE1FF] dark:border-[#2C3C56]"
             />
-          </div>
-          <div className="mb-6 flex md:mb-12 md:justify-end">
-            <Button
-              buttonColor="bg-blue-light dark:bg-blue-dark rounded-[9000px] py-2"
-              textColor="text-white-900 font-semibold"
-              hPadding="px-4"
-              width={isMobile ? "w-[345px]" : "w-[186px]"}
-              height={isMobile ? "h-[47px]" : "h-[69px]"}
-              extraStyles={isMobile ? "text-14" : "text-18"}
-              onClick={() => {}}
-              text="Send"
+            <h3 className="mt-4 text-18 dark:text-white-900 md:mt-6 md:text-20">
+              What is your email?
+            </h3>
+            <Input
+              type="email"
+              name="user_email"
+              placeholder=""
+              onChange={() => {}}
+              width={isMobile ? "327px" : "763px"}
+              height={isMobile ? "46px" : "56px"}
+              inputColor="bg-white-800 dark:bg-black-300 border-solid border-[1px] border-[#CCE1FF] dark:border-[#2C3C56]"
             />
-          </div>
+            <h3 className="mt-4 text-18 dark:text-white-900 md:mt-6 md:text-20">
+              Write something about your project goals and timeframe
+            </h3>
+            <textarea
+              placeholder=""
+              name="message"
+              onChange={() => {}}
+              className={`${isMobile ? "w-[327px]" : "w-[763px]"} h-[190px] ${
+                isMobile ? "px-4 py-2" : ""
```

Comment:
```suggestion
              className={`${isMobile ? "w-[327px] px-4 py-2" : "w-[763px]"} h-[190px] 
```

Can't you combine these? or does it not work?